keyboards/gh60/gh60.h

```diff
@@ -88,8 +88,24 @@ inline void gh60_wasd_leds_off(void)   	{ DDRF &= ~(1<<7); PORTF &= ~(1<<7); }
     { K40, K41, K42, KC_NO,    KC_NO,    K45, KC_NO,    KC_NO,    KC_NO,    K49, K4A, K4B, K4C, K4D }  \
 }
 
-#define LAYOUT_60_ansi KEYMAP_ANSI
-#define LAYOUT_60_iso KEYMAP_ISO
-#define LAYOUT_60_ansi_split_bs_rshift KEYMAP_HHKB
+/* ANSI with split Right Shift. No extra keys for ISO */
+#define LAYOUT_60_ansi_split_rshift( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
+    K30, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B,      K3D, K3C, \
+    K40, K41, K42,           K45,                     K4A, K4B, K4C, K4D  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, KC_NO,    K2D }, \
+    { K30, KC_NO,    K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D }, \
+    { K40, K41, K42, KC_NO,    KC_NO,    K45, KC_NO,    KC_NO,    KC_NO,    KC_NO, K4A, K4B, K4C, K4D }  \
+}
+
+#define KEYMAP LAYOUT
```

Comment:
Are these #defines really necessary?

Reply:
I left them this way because for some of the keymaps, the updated matrix names are vastly different from the original names, the `KEYMAP_HHKB`/`LAYOUT_60_ansi_split_bs_rshift` especially. (The matrix itself wasn't even for an HHKB layout because it has a full bottom row.) I wanted the users who contributed the keymaps to still recognize them moving forward.

I can change this if it's really needed.